homeassistant/components/ipma/config_flow.py

```diff
@@ -18,6 +18,13 @@ def __init__(self):
         """Init IpmaFlowHandler."""
         self._errors = {}
 
+    async def async_step_import(self, config):
```

Comment:
What will import a config here? The platform setup is removed and there's no `async_setup` in the init module.

Reply:
You are absolutely right

Checking history it hasn't been possible to use YAML since 2019, so I created https://github.com/home-assistant/core/pull/77674 to cleanup this